### PR TITLE
Update meshery-ui doc contribution flow

### LINKED ISSUE #7339: [Docs] Update contribution doc Meshery UI
Update the Contributing to [Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui) section of the doc with following details.

#### Current State
Make sure to have Meshery Server configured, up and running on the default port `http://localhost:9081` before proceeding to access and work on the UI server at `http://localhost:3000`.

#### Desired State
Make sure to have Meshery Server configured, up and running on the default port `http://localhost:9081` **and login with meshery provider (go to the `http://localhost:9081` to login with meshery provider)** before proceeding to access and work on the UI server at `http://localhost:3000`.

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by senali-d**
**Notes for Reviewers**

This PR fixes #7339 

Here's how it looks:

image

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit bfe4e9fb54a9996a831ca07ad6dda523199ccad6 at: https://640f28f1da84f4022fb4a8de--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/contributing-ui.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ If you want to work on the UI, it will be a good idea to use the included UI dev
 
 Refer to [Contributing to Meshery Server](contributing-server), if needed.
 
-> Make sure to have Meshery Server configured, up and running on the default port `http://localhost:9081` before proceeding to access and work on the UI server at `http://localhost:3000`.
+> Make sure to have Meshery Server configured, up and running on the default port `http://localhost:9081` and login with meshery provider (go to the `http://localhost:9081` to login with meshery provider) before proceeding to access and work on the UI server at `http://localhost:3000`.
```
Comment (leecalcote) [inline docs/pages/project/contributing/contributing-ui.md:130]:
```suggestion
> Make sure to have Meshery Server configured, up and running on the default port `http://localhost:9081` and choose a provider to login with (visit `http://localhost:9081`) before proceeding to access and work on the UI server at `http://localhost:3000`.
```

**Comment by l5io** [comment]
🚀 Preview for commit 2cae75cfdf778feb2d3fd6fc0191e3df759ebd32 at: https://640fe1e2c9f3fe1ec9b914ce--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
Good stuff, @senali-d 👏

**Comment by leecalcote** [comment]
yay, @senali-d! 🎈

**Pull request merged**